intel_realsense_payload_for_spotsim2real/README.md

```diff
@@ -0,0 +1,144 @@
+<!--
+Copyright (c) 2023 Boston Dynamics, Inc.  All rights reserved.
+
+Downloading, reproducing, distributing or otherwise using the SDK Software
+is subject to the terms and conditions of the Boston Dynamics Software
+Development Kit License (20191101-BDSDK-SL).
+-->
+
+# Intel RealSense Image Service With Depth 
+
+This code is a modified version of the [web-cam-service example](https://github.com/boston-dynamics/spot-sdk/tree/master/python/examples/web_cam_image_service) by Boston Dynamics Spot-SDK. This implements the standard Boston Dynamics API image service and communicates to Intel RealSense Camera using pyrealsense2 python package.
+Contrary to this [web-cam-service example](https://github.com/boston-dynamics/spot-sdk/tree/master/python/examples/web_cam_image_service)
+our modified service can also send depth images along with rgb images.
+The `startpayload.sh` will run the `intelrealsense_image_service.py` locally. The image service is an implementation of the GRPC image service, which has the ListImageSources RPC and the GetImage RPC. The service can encode information from the Intel camera connection into a protobuf message.
+   
+## Prerequisites
+
+#### Hardware Setup:
+![](images_for_readme/spot_core_intel_realsense.drawio.png)
+Connect spot core I/O on Spot's rear payload port.<br> 
+Unscrew the black port cap protruding from its back & attach Spot GXP interface. This will have ethernet port to plug RJ45 cable & power cable coming out of the GXP that should power the SPOT CORE. <strong>Spot core will have two ethernet ports & innermost ethernet port should be connect with GXP using external RJ45 cable. This order is mandatory to work.</strong> <br>
```

Comment:
Add a message that we need to ensure that both ports are covered by either spot core or cap. We cannot leave one port open, otherwise we cannot boot Spot and spot core.